src/django_watchfiles/__init__.py

```diff
@@ -62,12 +62,15 @@ def file_filter(self, change: watchfiles.Change, filename: str) -> bool:
             # print("Path in watched files")
             return True
         for directory, globs in self.directory_globs.items():
-            if path.is_relative_to(directory):
+            try:
+                relative_path = path.relative_to(directory)
```

Comment:
please put the remaining lines under an `else` clause

https://adamj.eu/tech/2019/10/02/limit-your-try-clauses-in-python/

Reply:
TIL thanks! Updated and re-tested.